src/Directory.Build.props

```diff
@@ -18,8 +18,8 @@
     <LangVersion>11.0</LangVersion>
   </PropertyGroup>
   <PropertyGroup>
-    <XunitVersion>2.5.3</XunitVersion>
-    <XunitRunnerVersion>2.5.3</XunitRunnerVersion>
+    <XunitVersion>2.8.1</XunitVersion>
```

Comment:
Redundant - we already use these newer versions on `dev` but I had to capture what the logs originally should have looked like on v1.5.20, where we had not yet performed these upgrades. Verify.Xunit required the newer versions.